x-pack/plugins/observability_solution/apm/server/routes/dependencies/get_top_dependency_spans.ts

```diff
@@ -134,10 +133,10 @@ export async function getTopDependencySpans({
       },
       body: {
         track_total_hits: false,
-        size: transactionIds.length,
+        size: traceIds.length,
         query: {
           bool: {
-            filter: [...termsQuery(TRANSACTION_ID, ...transactionIds)],
+            filter: [...termsQuery(TRACE_ID, ...traceIds), { exists: { field: TRANSACTION_ID } }],
```

Comment:
Should we have the TRACE_ID here instead?

Reply:
we want to make sure that for those trace ids coming from the spans transaction.id exist

Reply:
Ah Ok, it makes sense - thanks for explaining!